### PR TITLE
added some good shit

### PR DESCRIPTION
**Pull request by AmyDZeng**

### CODE REVIEW

File: content/W2016/W16-good-shit.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Title: Yee
+Date: 2016-03-15
+Category: Events
+Tags: panels, event
+Slug: coop-panel-w16
+Author: Amy Zeng
```
Comment (AmyDZeng) [inline content/W2016/W16-good-shit.md]: das me

**Comment by fboxwala** [comment]
Hi Amy - Can you make a commit message as per the guidelines on the Feature development page?

File: content/W2016/W16-good-shit.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+Summary: code stuff
+
+Jobmine opens this Saturday, and that is kind of nerve-wracking. To help you
+gear up for the co-op season, WiCS is holding a career panel this upcoming 
+Monday! Join us and hear the experiences of some awesome women in tech at
+UWaterloo, and bring whatever questions you have about co-op, jobs and jobmine!
+
```
Comment (fboxwala) [inline content/W2016/W16-good-shit.md]: Can you change this and add some information for your own event

File: content/W2016/W16-good-shit.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+Title: Yee
```
Comment (fboxwala) [inline content/W2016/W16-good-shit.md]: "Yee" is not a great name for an event :)

Comment (Gibstick) [inline content/W2016/W16-good-shit.md]: Why did you add me to this repo dammit Fatema
